src/components/VCalendar/VCalendar.ts

```diff
@@ -0,0 +1,34 @@
+// Styles
+// import '../../stylus/components/_calendar-daily.styl'
+
+// Types
+import { VNode } from 'vue'
+
+// Mixins
+import CalendarWithIntervals from './mixins/calendar-with-intervals'
+
+/* @vue/component */
+export default CalendarWithIntervals.extend({
+  name: 'v-calendar',
+
+  computed: {
+    classes (): object {
```

Comment:
should return ClassesObject type

Reply:
Looks like that type is not defined in the current dev branch